server/src/main/scala/drt/server/feeds/bhx/BHXForecastFeed.scala

```diff
@@ -0,0 +1,36 @@
+package drt.server.feeds.bhx
+
+import akka.NotUsed
+import akka.actor.{ActorSystem, Cancellable}
+import akka.stream.scaladsl.Source
+import com.typesafe.config.ConfigFactory
+import drt.shared.Arrival
+import org.slf4j.{Logger, LoggerFactory}
+import scala.collection.immutable.Seq
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+
+object BHXForecastFeed extends BHXFeedConfig {
+  override val log: Logger = LoggerFactory.getLogger(getClass)
+  override val endPointUrl: String = ConfigFactory.load.getString("feeds.bhx.soap.endPointUrl")
```

Comment:
we usually pass all config values in from the controller - referencing config values here means that tests will be reading config files which can lead to errors.